server/src/cg/group4/server/database/query/GetGroup.java

```diff
@@ -22,28 +22,30 @@
      *
      * @param groupId The given group id.
      */
-    public GetGroup(String groupId) {
+    public GetGroup(final String groupId) {
         cGroupId = groupId;
     }
 
     @Override
-    public Group query(DatabaseConnection databaseConnection) throws SQLException {
+    public Group query(final Connection databaseConnection) throws SQLException {
+        String preparedStatement = "SELECT * FROM 'Group' INNER JOIN User ON 'Group'.OwnerId = User.Id "
```

Comment:
Owner data is now stored on the local db. So this wont work... remote shouldn't even have a user table.. Instead groups should contain both the owners id and name.